weave/tests/test_index_sqlite.py

```diff
@@ -149,3 +164,33 @@ def test_index_uploaded_basket_not_found_in_another_index(test_pantry):
     # Remove the .db files that are not cleaned up with 'test_pantry'
     os.remove(pantry_1.index.db_path)
     os.remove(pantry_2.index.db_path)
+
+
+def test_index_sqlite_track_basket_adds_to_parent_uuids(test_index):
```

Comment:
I understand why you put this here, but would it also make sense to add a test to the test_index file? Because I think the problem was we set it up correctly in generate_index, but not in track_basket. So maybe we need an index test that only does track basket, then check for parents.